tests/roles/dataplane_rhev_adoption/tasks/main.yaml

```diff
@@ -0,0 +1,328 @@
+      - name: InternalApi
+        dnsDomain: internalapi.redhat.local
+        subnets:
+        - name: subnet1
+          allocationRanges:
+          - end: 172.15.2.15
+            start: 172.15.2.4
+          cidr: 172.15.2.0/24
+          vlan: 20
+      - name: External
+        dnsDomain: external.redhat.local
+        subnets:
+        - name: subnet1
+          allocationRanges:
+          - end: 10.4.0.15
+            start: 10.4.0.4
+          cidr: 10.4.0.0/24
+          gateway: 10.4.0.1
+          vlan: 10
+      - name: Tenant
+        dnsDomain: tenant.redhat.local
+        subnets:
+        - name: subnet1
+          allocationRanges:
+          - end: 172.15.0.15
+            start: 172.15.0.4
+          cidr: 172.15.0.0/24
+          vlan: 50
+    EOF
+
+- name: Slurp the private key
+  no_log: "{{ use_no_log }}"
+  ansible.builtin.slurp:
+    path: "{{ edpm_privatekey_path }}"
+  register: edpm_privatekey
+  when: edpm_encoded_privatekey is undefined
+
+- name: create dataplane-adoption-secret.yaml
+  no_log: "{{ use_no_log }}"
+  ansible.builtin.shell: |
+    {{ shell_header }}
+    {{ oc_header }}
+    oc apply -f - <<EOF
+    apiVersion: v1
+    kind: Secret
+    metadata:
+        name: dataplane-adoption-secret
+    data:
+        ssh-privatekey: "{{ edpm_encoded_privatekey|default(edpm_privatekey.content) }}"
+    EOF
+
+- name: "[hack] create nova-metadata-neutron-config-secret.yaml"
+  no_log: "{{ use_no_log }}"
+  ansible.builtin.shell: |
+    {{ shell_header }}
+    {{ oc_header }}
+    oc apply -f - <<EOF
+    apiVersion: v1
+    kind: Secret
+    metadata:
+        name: nova-metadata-neutron-config
+    data:
+        05-nova-metadata.conf: |
+    $(echo "[DEFAULT]\nnova_metadata_host = 1.2.3.4\nnova_metadata_port = 8775\nnova_metadata_protocol = http\nmetadata_proxy_shared_secret = 1234567842\n" | base64 | sed 's/^/        /')
+    EOF
+
+- name: set Nova copy shell vars
+  no_log: "{{ use_no_log }}"
+  ansible.builtin.set_fact:
+    nova_shell_vars: |
+      CONTROLLER1_SSH="{{ controller1_ssh }}"
+      CONTROLLER2_SSH="{{ controller2_ssh }}"
+      CONTROLLER3_SSH="{{ controller3_ssh }}"
+      COMPUTE_SSH="{{ compute_ssh }}"
+
+- name: stop nova services on controllers
+  no_log: "{{ use_no_log }}"
+  ansible.builtin.shell: |
+    {{ shell_header }}
+    {{ oc_header }}
+    {{ nova_shell_vars }}
+
+    ServicesToStop=("tripleo_nova_api_cron.service"
+                    "tripleo_nova_api.service"
+                    "tripleo_nova_conductor.service"
+                    "tripleo_nova_metadata.service"
+                    "tripleo_nova_vnc_proxy.service")
+
+    echo "Stopping nova services"
+
+    for service in ${ServicesToStop[*]}; do
+        echo "Stopping the service: $service in each controller node"
+        $CONTROLLER1_SSH sudo systemctl stop $service
+        $CONTROLLER2_SSH sudo systemctl stop $service
+        $CONTROLLER3_SSH sudo systemctl stop $service
+    done
+
```

Comment:
actually, this is outdated: https://github.com/openstack-k8s-operators/data-plane-adoption/blob/main/tests/roles/dataplane_adoption/tasks/main.yaml#L111-L155

Reply:
So, should i remove 92L-136L form this file?

Reply:
actually you have to replace 92L-160L with
```yaml
- name: generate an ssh key-pair nova-migration-ssh-key secret
  no_log: "{{ use_no_log }}"
  ansible.builtin.shell: |
    {{ shell_header }}
    {{ oc_header }}
    cd "$(mktemp -d)"
    ssh-keygen -f ./id -t ed25519 -N ''
    oc get secret nova-migration-ssh-key || oc create secret generic nova-migration-ssh-key \
    -n openstack \
    --from-file=ssh-privatekey=id \
    --from-file=ssh-publickey=id.pub \
    --type kubernetes.io/ssh-auth
    rm -f id*
    cd -

- name: create a Nova Compute Extra Config service
  no_log: "{{ use_no_log }}"
  ansible.builtin.shell: |
    {{ shell_header }}
    {{ oc_header }}
    oc apply -f - <<EOF
    apiVersion: v1
    kind: ConfigMap
    metadata:
      name: nova-compute-extraconfig
      namespace: openstack
    data:
      19-nova-compute-cell1-workarounds.conf: |
        [workarounds]
        disable_compute_service_check_for_ffu=true
    ---
    apiVersion: dataplane.openstack.org/v1beta1
    kind: OpenStackDataPlaneService
    metadata:
      name: nova-compute-extraconfig
      namespace: openstack
    spec:
      label: nova.compute.extraconfig
      configMaps:
        - nova-compute-extraconfig
      secrets:
        - nova-cell1-compute-config
        - nova-migration-ssh-key
      playbook: osp.edpm.nova
    EOF
```